app/src/PeopleList/index.js

```diff
@@ -4,11 +4,21 @@ import styles from './styles.css';
 /**
  * This component lists the users that will show up for selection.
  */
+const people = [{
+  name: 'Clay'
+}, {
+  name: 'Elias'
+}, {
+  name: 'Sterling'
+}, {
+  name: 'Andy'
+}];
+
 const PeopleList = props => (
   <ul className={styles.root}>
-    {props.people.map(p => (
-      <li key={p.name}>{p.name}</li>
-    ))}
+    {people.map(p => ( //I changed this from props.people.map to people.map, because I didn't understand how to make the former work
```

Comment:
Yeah I'm trying out react-router v4 to see how I like it... I'm not sure how that works either... I'll have to read some more.